test/e2e_node/device_plugin_test.go

```diff
@@ -714,11 +714,13 @@ func testDevicePluginNodeReboot(f *framework.Framework, pluginSockDir string) {
 
 		ginkgo.BeforeEach(func(ctx context.Context) {
 			ginkgo.By("Wait for node to be ready")
-			gomega.Eventually(ctx, func(ctx context.Context) bool {
-				nodes, err := e2enode.TotalReady(ctx, f.ClientSet)
-				framework.ExpectNoError(err)
-				return nodes == 1
-			}, time.Minute, time.Second).Should(gomega.BeTrue())
+			gomega.Eventually(ctx, func(ctx context.Context) error {
+				_, err := e2enode.TotalReady(ctx, f.ClientSet)
+				if err != nil {
+					return err
+				}
+				return nil
+			}, time.Minute, time.Second).Should(gomega.Succeed())
```

Comment:
Doesn't this remove the `nodes == 1` check?

Just return both values, then do:

```suggestion
			}, time.Minute, time.Second).Should(gomega.Equal(1), "one node should be ready")
```
```